test/e2e/tests/test_subscription.py

```
@@ -108,19 +106,16 @@ def _decode_jwt_payload(token: str) -> Optional[dict]:
 
 def _get_cluster_token():
+    """Get OC token for API key management operations (not for inference)."""
     sa_ns = os.environ.get("E2E_TEST_TOKEN_SA_NAMESPACE")
     sa_name = os.environ.get("E2E_TEST_TOKEN_SA_NAME")
     if sa_ns and sa_name:
         token = _create_sa_token(sa_name, namespace=sa_ns)
     else:
-        token_result = subprocess.run(
-            ["oc", "whoami", "-t"], capture_output=True, text=True
-        )
+        token_result = subprocess.run(["oc", "whoami", "-t"], capture_output=True, text=True)
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)